modules/special-rules/pages/save.adoc

```diff
@@ -1,2 +1,12 @@
 = Save
 
+Due to exceptionally resilient armour, protective fields, or extreme skills and toughness, these troops are capable of surviving almost any impact.
+
+---
+
+Apply these rules to units with the {save} ability:
+
+* Each time the unit suffers a hit, from any source, roll a die. On a roll of 4+, the hit is ignored and causes no damage.
+* The {save} ability applies to attacks that ignore void shields and power fields.
```

Comment:
Just spotted that the Q&A for Save allows it to be used against destruction via your Transport vehicle dying, even though that is **not** a hit.

This is just rough text to act as a placeholder/reminder.

```suggestion
* The {save} ability applies to attacks that ignore void shields and power fields.
* The {save} ability applies to transported units when their {transport} vehicle is destroyed.
```

Reply:
Hmm. Looking at this again, I think it would be cleaner for the Transport rules to say that embarked units all suffer a hit when the Transport is destroyed. I'll post on FB to see if anyone can spot rules issues with that approach.

If destroyed Transports result in a 'hit' on the embarked units, this added bullet point can simply be left out. _Maybe_ adding a tip or note as a reminder instead.

Reply:
The Transport page has been updated to say that the units suffer hits, as per the FB discussions. So this is suggestion is no longer needed.